contrib/go.mongodb.org/mongo-driver.v2/mongo/option.go

```diff
@@ -0,0 +1,67 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package mongo
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/namingschema"
+)
+
+const defaultServiceName = "mongo"
+
+type config struct {
+	serviceName   string
+	spanName      string
+	analyticsRate float64
+}
+
+// Option represents an option that can be passed to Dial.
+type Option func(*config)
+
+func defaults(cfg *config) {
+	cfg.serviceName = namingschema.ServiceNameOverrideV0(defaultServiceName, defaultServiceName)
+	cfg.spanName = namingschema.OpName(namingschema.MongoDBOutbound)
```

Comment:
this package `namingschema` does not exist anymore, you can check in the existing mongodb integration how this is done in v2.